web/templates/change_email_modal.hbs

```diff
@@ -1,4 +1,4 @@
 <form id="change_email_form">
     <label for="email" class="modal-field-label">{{t "New email" }}</label>
-    <input type="text" name="email" class="modal_text_input" value="{{delivery_email}}" autocomplete="off" spellcheck="false" autofocus="autofocus"/>
+    <input id="email" type="text" name="email" class="modal_text_input" value="{{delivery_email}}" autocomplete="off" spellcheck="false" autofocus="autofocus"/>
```

Comment:
This ID is too general -- seems like it could overlap with some other field elsewhere. Let's rename it to something like `change-email-form-input-email`?